tmk_core/protocol/arm_atsam/main_arm_atsam.c

```diff
@@ -305,6 +305,10 @@ int main(void) {
             // dprintf("5v=%u 5vu=%u dlow=%u dhi=%u gca=%u gcd=%u\r\n", v_5v, v_5v_avg, v_5v_avg - V5_LOW, v_5v_avg - V5_HIGH, gcr_actual, gcr_desired);
         }
 #endif  // CONSOLE_ENABLE
+
+        // Run housekeeping
+        housekeeping_task_kb();
+        housekeeping_task_user();
```

Comment:
The `arm_atsam` implementation calls `keyboard_task();` in it's loop, thus the housekeeping calls should be covered by that.

Reply:
Good catch; if you can raise a PR for it, it would be wonderful.